packages/circuit-ui/components/Dialog/Dialog.spec.tsx

```diff
@@ -47,6 +47,20 @@ describe('Dialog', () => {
 
   let originalHTMLDialogElement: typeof window.HTMLDialogElement;
 
+  function swipeDown(dialog: Element) {
+    fireEvent.touchStart(dialog, {
+      targetTouches: [{ clientX: 0, clientY: 0 }],
+    });
+
+    fireEvent.touchMove(dialog, {
+      targetTouches: [{ clientX: 0, clientY: 700 }],
+    });
+
+    fireEvent.touchEnd(dialog, {
+      changedTouches: [{ clientX: 0, clientY: 300 }],
+    });
```

Comment:
Should the end distance match the move distance? 

```suggestion
    fireEvent.touchMove(dialog, {
      targetTouches: [{ clientX: 0, clientY: 700 }],
    });

    fireEvent.touchEnd(dialog, {
      changedTouches: [{ clientX: 0, clientY: 700 }],
    });
```